core/cmd/shell_local.go

```diff
@@ -223,6 +223,29 @@ func initLocalSubCmds(s *Shell, safe bool) []cli.Command {
 						},
 					},
 				},
+				{
+					Name:    "delete-chain",
+					Aliases: []string{},
+					Usage:   "Commands for cleaning up chain specific db tables. WARNING: This will ERASE ALL chain specific data referred to by --chaintype and --chainid options for the specified database, referred to by CL_DATABASE_URL env variable or by the Database.URL field in a secrets TOML config.",
+					Action:  s.CleanupChainTables,
+					Before:  s.validateDB,
+					Flags: []cli.Flag{
+						cli.IntFlag{
+							Name:     "chainid",
+							Usage:    "chainID based on which table cleanup will be done",
```

Comment:
I think we prefer this format:
```suggestion
							Name:     "chain-id",
							Usage:    "chainID based on which table cleanup will be done",
```
Also, a case could be made for just `id` :thinking: